django/test/testcases.py

```diff
@@ -825,6 +825,10 @@ class TransactionTestCase(SimpleTestCase):
     # are not available, we allow queries to be run.
     allow_database_queries = True
 
+    @classmethod
```

Comment:
Oh, it's fine to take out now. I agree with your reasoning. Did you want me to do that? I saw "ready for checkin" on the ticket so thought that was a sign that the patch didn't need further updates.